.jenkins/pytorch/build-asan.sh

```diff
@@ -32,7 +32,7 @@ export ASAN_OPTIONS=detect_leaks=0:symbolize=1:detect_odr_violation=0
 # TODO: Make the ASAN flags a centralized env var and unify with USE_ASAN option
 CC="clang" CXX="clang++" LDSHARED="clang --shared" \
   CFLAGS="-fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all -shared-libasan -pthread" \
-  CXX_FLAGS="-pthread" \
+  CXXFLAGS="-fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all -shared-libasan -pthread" \
```

Comment:
Specifying any of the options shouldn't be necessary, as `USE_ASAN` CMakeList option should to the trick